spring-xd-shell/src/test/java/org/springframework/xd/shell/command/AbstractStreamIntegrationTest.java

```diff
@@ -28,17 +28,7 @@
 import org.springframework.jdbc.datasource.SimpleDriverDataSource;
 import org.springframework.xd.shell.AbstractShellIntegrationTest;
 import org.springframework.xd.shell.command.fixtures.HttpSource;
-import org.springframework.xd.test.fixtures.Disposable;
-import org.springframework.xd.test.fixtures.FileSink;
-import org.springframework.xd.test.fixtures.FileSource;
-import org.springframework.xd.test.fixtures.JdbcSink;
-import org.springframework.xd.test.fixtures.JdbcSource;
-import org.springframework.xd.test.fixtures.MailSink;
-import org.springframework.xd.test.fixtures.NonPollingImapSource;
-import org.springframework.xd.test.fixtures.PollingMailSource;
-import org.springframework.xd.test.fixtures.TailSource;
-import org.springframework.xd.test.fixtures.TcpSink;
-import org.springframework.xd.test.fixtures.TcpSource;
+import org.springframework.xd.test.fixtures.*;
```

Comment:
Nope, but I switched laptops :) Will fix

Warning, one of your favorite discussions ahead: Why aren't the formatting options into VCS again?